src/main/java/com/fuem/repositories/FeedbackDAO.java

```diff
@@ -0,0 +1,48 @@
+package com.fuem.repositories;
+
+import com.fuem.models.Feedback;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+public class FeedbackDAO extends SQLDatabase {
+    
+    private static final Logger logger = Logger.getLogger(FeedbackDAO.class.getName());
+    
+    // SQL truy vấn để chèn phản hồi vào bảng Feedback
+    private static final String INSERT_FEEDBACK = 
+            "INSERT INTO Feedback (guestId, eventId, content) VALUES (?, ?, ?)";
+
+ 
+    public boolean saveFeedback(Feedback feedback) {
+        try (Connection conn = getConnection(); // Lấy kết nối từ phương thức getConnection()
+             PreparedStatement ps = conn.prepareStatement(INSERT_FEEDBACK)) {
+            
+            ps.setInt(1, (int) feedback.getGuestId());  // Thiết lập guestId
+            ps.setInt(2, (int) feedback.getEventId());  // Thiết lập eventId
+            ps.setString(3, feedback.getContent()); // Thiết lập nội dung phản hồi
+            
+            return ps.executeUpdate() > 0; // Trả về true nếu đã lưu thành công
+        } catch (SQLException e) {
+            logger.log(Level.SEVERE, "Error saving feedback", e);
+            return false; // Trả về false nếu có lỗi
+        }
+    }
```

Comment:
- Dùng hàm tớ làm chứ cậu, tự set chi cực
```suggestion
    public boolean saveFeedback(Feedback feedback) {
        int rowChange = 0;
        try (Connection conn = getConnection(); // Lấy kết nối từ phương thức getConnection();) {
            rowChange = executeUpdateQueryPreparedStatement(conn, INSERT_FEEDBACK, feedback.getGuestId(), feedback.getEventId(), feedback..getContent();
        } catch (SQLException e) {
            logger.log(Level.SEVERE, "Error saving feedback", e);
            return false; // Trả về false nếu có lỗi
        }
        
        return (rowChange > 0);
    }
```